src/v/cloud_storage/materialized_resources.cc

```diff
@@ -144,8 +144,9 @@ get_throughput_limit(std::optional<size_t> device_throughput) {
     }
 
     auto tp = std::min(hard_limit, device_throughput.value());
+    constexpr auto tput_overshoot_frac = 16;
```

Comment:
Is there anything special about this number? Or was it empirically determined?

If empirically determined, how did you determine how effective this PR is (in case we need to adjust it further down the line)?

Reply:
it's a simple back of the envelope calculation (2GiB/s with 50% allowance for TS will give us around 60MiB/s per node)
we only need to limit disk as a last resort, the code is relying on network limit which is set per shard

Reply:
> it's a simple back of the envelope calculation (2GiB/s with 50% allowance for TS will give us around 60MiB/s per node)
we only need to limit disk as a last resort, the code is relying on network limit which is set per shard

add to commit message?